File: remote-debugging-spring-boot-application/src/main/java/de/rieckpil/blog/Book.java
package de.rieckpil.blog;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.Data;

@Data
@Entity
public class Book {

  @Id @GeneratedValue private Long id;

  @Column(nullable = false, unique = true)
  private String isbn;

  @Column(nullable = false)
  private String author;

  @Column(nullable = false)
  private String title;
}

File: java-ee-with-kotlin/src/main/kotlin/de.rieckpil.blog/Book.kt
package de.rieckpil.blog

import javax.persistence.*

@Entity
@Table(name = "books")
data class Book(
        @Id
        @GeneratedValue
        var id: Long?,

        @Column(nullable = false, unique = true)
        val title: String,

        @Column(nullable = false)
        val author: String) {
}